tests/components/axis/test_device.py

```diff
@@ -113,10 +128,10 @@ async def test_device_setup(hass):
     assert forward_entry_setup.mock_calls[1][1] == (entry, "binary_sensor")
     assert forward_entry_setup.mock_calls[2][1] == (entry, "switch")
 
-    assert device.host == ENTRY_CONFIG[axis.CONF_HOST]
-    assert device.model == ENTRY_CONFIG[axis.device.CONF_MODEL]
-    assert device.name == ENTRY_CONFIG[axis.device.CONF_NAME]
-    assert device.serial == ENTRY_CONFIG[axis.CONF_MAC]
+    assert device.host == ENTRY_CONFIG[CONF_HOST]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)